File: javaNetbeans/CollectionReal/src/listconcept/MaxMin.java
package listconcept;

import java.util.ArrayList;
import java.util.Scanner;

public class MaxMin {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter size of array: ");
        int size = sc.nextInt();
        
        ArrayList<Integer> arr = new ArrayList<>(size);
        System.out.println("Enter "+size +" element: ");
        for(int i = 0; i < size; i++){
            int value = sc.nextInt();
            arr.add(value);
        }
        
        System.out.println("Default array element is: ");
        for(int i = 0; i < arr.size(); i++){
            System.out.print(arr.get(i)+" ");
        }
        
        int min = arr.get(0);
        int max = arr.get(0);
        
        for(int i = 0; i < arr.size(); i++){
            if(min > arr.get(i)){
                min = arr.get(i);
            }else{
                max = arr.get(i);
            }
        }
        
        System.out.println("Mininum value is: "+min);
        System.out.println("Maximum value is: "+max);
    }
}

File: MeetingWithJS/FINAL_JS_REVESION/JS/Array22222/MaxMin.js
// (function (){
//     let max = -Infinity, min = Infinity;
//     let arr = [1,2,34,5,6,7,9];
    
//     for(let value of arr){
//         if(value > max){
//             max = value;
//         }
//         if(value < min){
//             min = value;
//         }
//     }
//     console.log(max, min)
// })();

(function (){
    let max = -Infinity, min = Infinity;
    let arr = [3,4,5,6,7,8];
    arr.forEach((element)=>{
        if(element > max){
            max = element;
        }
         if(element < min){
            min = element;
        }
    })
    console.log(max+" "+ min);
})();